wsmaster/che-core-api-devfile/src/main/java/org/eclipse/che/api/devfile/server/convert/tool/dockerimage/DockerimageToolProvisioner.java

```diff
@@ -127,6 +135,15 @@ public void provision(Devfile devfile, WorkspaceConfigImpl workspaceConfig)
     devfile.getTools().add(dockerimageTool);
   }
 
+  private EntryPoint toEntryPoint(MachineConfig machineConfig) throws WorkspaceExportException {
+    try {
+      EntryPointParser entryPointParser = new EntryPointParser();
```

Comment:
maybe put the parser into a field ?

Reply:
Also, it may be injected to make it possible to test `DockerimageToolProvisioner` functionality with mocked implementation to avoid testing two classes functionalities by one test class

Reply:
`EntryPointParser` now mocked in the tests.